src/ima.js

```diff
@@ -1041,6 +1041,8 @@ class Ima extends BasePlugin implements IMiddlewareProvider, IAdsControllerProvi
     this._contentComplete = true;
     if (this._currentAd && !this._currentAd.isLinear()) {
       this.reset();
+    } else if (!(this._adsManager && this._adsManager.getCuePoints().includes(-1))) {
```

Comment:
Can't we make it earlier (in `_requestAds` for example)? or we must wait to media ended?

Reply:
earlier need to check there is no midroll forward. 
And what if the user seeks forward and backward.
I prefer to avoid these situations.